conf/parse_test.go

```diff
@@ -143,6 +143,32 @@ func TestEnvVariableStringStartingWithNumberUsingQuotes(t *testing.T) {
 	test(t, fmt.Sprintf("foo = $%s", evar), ex)
 }
 
+func TestEnvVariableURL(t *testing.T) {
+	cluster := `
+		cluster {
+			# set the variable token
+			TOKEN: abc
+			authorization {
+				user: user
+				password: $TOKEN
```

Comment:
one side effect from this change is that now env vars within quotes would also work, so these would be the same now:

```
password: $TOKEN
password: "$TOKEN"
```

thinking that there might be some edge cases which could be introduced potentially where an env var is replaced by accident on a string, for example if `SYS` for some reason is defined this could be replaced in permissions:

```
allow = ["$SYS"]
```

Reply:
Let me think about some of those, good points.